src/main/java/Duke.java

```diff
@@ -1,10 +1,60 @@
+import java.io.IOException;
+import java.io.FileNotFoundException;
+
+import duke.DukeIO;
+import duke.InvalidCommandException;
+import duke.MyDuke;
+import duke.Parser;
+import duke.Storage;
+import duke.TaskList;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    private static MyDuke duke = new MyDuke();
+    private static DukeIO dukeIo = new DukeIO();
+    private static Storage storage = new Storage();
+    private static TaskList taskList = TaskList.ofNull();
+    private static Parser parser = new Parser();
+
+    public static void main(String[] args) 
+            throws InvalidCommandException, IOException, ClassNotFoundException {
+        
+        duke.init();
+
+        try {
+            taskList.loadFrom(storage.load());
+        } catch (FileNotFoundException p) {
+            dukeIo.echoMessage("Nothing to load");
+        }
```

Comment:
What would happen if the file was present but contains invalid items? (partially or fully)